dcat/config.js

```diff
@@ -1,18 +1,18 @@
 var respecConfig = {
 //	preProcess: [dfn_index],
 //    subtitle: "Version 2",
-    specStatus: "ED",
+    specStatus: "FPWD",
```

Comment:
As suggested by @andrea-perego, the SpecStatus must not change in the GitHub draft.
```suggestion
    specStatus: "ED",
```